pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -76,6 +79,21 @@ public void writeResource(HapiRowDescriptor element)
             .setVersionId(element.resourceVersion())
             .setLastUpdated(simpleDateFormat.parse(element.lastUpdated()));
 
+    if (element.getTags() != null) {
+      List<Coding> tags = new ArrayList<>();
+      for (ResourceTag resourceTag : element.getTags()) {
+        // The column tag_type of value 0 means it's of type TAG, 1 for PROFILE and 2 for SYSTEM.
+        if (resourceTag.getTagType() == 1) {
+          CanonicalType canonicalType = new CanonicalType();
+          canonicalType.setValue(resourceTag.getCoding().getCode());
+          meta.setProfile(List.of(canonicalType));
+        } else if (resourceTag.getTagType() == 0) {
+          tags.add(resourceTag.getCoding());
+        }
+      }
+      meta.setTag(tags);
```

Comment:
Is there a difference in our output behavior if meta.tag is never set, vs. set to an empty list?

What happens to the output if there is a profile, but no FHIR tags?  In that case, we end up registering an empty list into meta.  Is that the desired output in that case?

e.g. is there

Reply:
no with empty list, parquet will not show anything for tags

<img width="541" alt="Screenshot 2023-03-15 at 12 11 01 PM" src="https://user-images.githubusercontent.com/105199873/225227413-62be3d27-ad77-4d29-9168-3732bbd4471f.png">

Reply:
Hmmm.  Still, I think for cleanliness, I think we should check if the list is populated before calling meta.setTag().  Sorry for not stating this before.

Reply:
Addressed this, thanks.